css/styles.css

```diff
@@ -50,3 +60,28 @@ body {
 .width {
   width: 100%;
 }
+
+.new-button {
+  width: 70px;
+  height: 35px;
+  background-color: #4c2676;
+  color: #fff;
+  font-size: larger;
+  font-family: 'Playfair Display', serif;
+  padding: 5px;
+  border-radius: 5px;
+}
+
+.align {
+  text-align: right;
+  margin-top: 1rem;
+}
+
+li {
+  list-style-type: none;
+  font-size: 20px;
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
This class is already provided for you in `layout.css`, so this isn't needed and can be deleted